protobuf/flow/entities/collection.proto

```diff
@@ -17,4 +17,5 @@ message CollectionGuarantee {
   bytes signature = 4;
   repeated bytes signer_ids = 5; // deprecated!! value will be empty. replaced by signer_indices
   bytes signer_indices = 6;
+  bytes chain_id = 7;
```

Comment:
this is the cluster chain id right? How about naming it `cluster_chain_id` so users don't confuse it with the network's `chain_id`